bookkeeper-dist/src/assemble/bin-all.xml

```diff
@@ -50,6 +50,33 @@
         <include>${basedir}/*.txt</include>
       </includes>
     </fileSet>
+    <fileSet>
+      <directory>../src/main/resources/deps</directory>
+      <outputDirectory>/deps</outputDirectory>
+      <includes>
+        <include>javax.servlet-api-3.1.0/CDDL+GPL-1.1</include>
+        <include>jsr-305/LICENSE</include>
+        <include>netty-3.10.1.Final/LICENSE.base64.txt</include>
```

Comment:
nit: can this simplified with `netty-3.10.1.Final/*`?

Reply:
done